AudibleApi/ApiHttpClient.cs

```diff
@@ -49,8 +49,7 @@ public override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage req
 
 		public static ApiHttpClient Create(HttpClientHandler innerHandler)
 		{
-			if (innerHandler is null)
-				throw new ArgumentNullException(nameof(innerHandler));
+			ArgumentValidator.EnsureNotNull(innerHandler, nameof(innerHandler));
```

Comment:
Ditto about replacing stuff like this client. They were probably over-engineered. Since they're so engrained though, replacing them would require careful testing.